src/main/java/io/github/thebusybiscuit/slimefun4/core/guide/unlockprovider/ExperienceUnlockProvider.java

```diff
@@ -0,0 +1,33 @@
+package io.github.thebusybiscuit.slimefun4.core.guide.unlockprovider;
+
+import javax.annotation.Nonnull;
+
+import org.bukkit.entity.Player;
+
+import io.github.thebusybiscuit.slimefun4.api.researches.Research;
+import io.github.thebusybiscuit.slimefun4.api.guide.SlimefunGuideUnlockProvider;
+import io.github.thebusybiscuit.slimefun4.implementation.Slimefun;
+
+/**
+ * This is default guide unlock provider research unlock used.
+ * Unlock research by withdrawing player's experience level.
+ *
+ * @see SlimefunGuideUnlockProvider
+ */
+public final class ExperienceUnlockProvider implements SlimefunGuideUnlockProvider {
```

Comment:
Is there a reason these are final?
It only prevents someone else from extending it which I feel like we _should_ allow